.github/workflows/go.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ master ]
+  workflow_call:
```

Comment:
If this is needed to
```
  go:
    name: Go
    uses: ./.github/workflows/go.yml
```

I would add a comment here. Because otherwise someone might clean this and we only realise that on release.

Reply:
Yup, per docs: https://docs.github.com/en/actions/sharing-automations/reusing-workflows#creating-a-reusable-workflow

> For a workflow to be reusable, the values for `on` must include `workflow_call`

I have pushed https://github.com/jacekkow/stayrtr/commit/4ca0110f70ff3be195d6141933e21bcb23bbc922